geoportailv3/static/js/draw/drawnfeaturesservice.js

```diff
@@ -133,6 +132,26 @@ app.DrawnFeatures.prototype.saveFeature = function(feature) {
 
 
 /**
+ * Copy anonymous features to mymaps
```

Comment:
Wouldn't it make more sense to rename the function to `moveAnonymousFeaturesToMymaps` as the features are "moved" rather than "copied"?